.evergreen/orchestration/configs/servers/basic-tls-ocsp-mustStaple.json

```diff
@@ -0,0 +1,17 @@
+{
+   "id" : "standalonenoauthssl",
+   "name": "mongod",
+   "procParams": {
+      "ipv6": true,
+      "bind_ip": "127.0.0.1,::1",
+      "logappend": true,
+      "journal": true,
+      "port": 27017
+   },
+   "sslParams": {
```

Comment:
I believe using the older parameter names is needed for maximum backwards compatibility.

Reply:
I believe that's correct, as the "tls" configuration API was introduced in [4.2](https://docs.mongodb.com/manual/tutorial/configure-ssl/#procedures-using-net-ssl-settings).